src/resources/ResourceSnapshots/ResourceSnapshotsInterfaces.ts

```diff
@@ -237,6 +239,12 @@ export interface SnapshotAccessModel {
     allowed: boolean;
 }
 
+export interface SnapshotListParams {
+    filter?: string;
+    direction?: SortingOrder;
```

Comment:
```suggestion
    direction?: SortingOrder;
```

The sources list uses `sortingOrder` and the snapshots list uses `direction`. Would it be possible to have the same names for all APIs?

Reply:
No pagination planned here?